fastq_filtrator.py

```diff
@@ -0,0 +1,98 @@
+
+# ### Функция для прочтения файла по 4 строки:
+
+def read_4lines(file):
+    lines = []
+    for i in range(4):
+        line = file.readline()
+        if line == "":
+            return []
+        lines.append(line.replace("\n", ""))
+    return lines
+
+
+
+# ###  Функция для проверки GC-состава:
+
+def gc_bounds_func(inter_gc, lines):
+    cnt_gc = 0
+    one_read = lines[1]
+    gc_count = (one_read.count('C') + one_read.count('G'))/len(one_read)*100
```

Comment:
Это хорошо читается, но не очень оптимально по производительности. count внутри себя делает полный проход по строке, и мы вызываем его два раза, хотя GC можно посчитать за один проход. Но это мелочи)

И ещё пробелы
```suggestion
    gc_count = (one_read.count('C') + one_read.count('G')) / len(one_read) * 100
```